triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/Jwt.scala

```diff
@@ -14,6 +15,7 @@ import com.daml.lf.integrationtest.CantonFixture
 import org.scalatest._
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AsyncWordSpec
+import scalaz.syntax.tag._
```

Comment:
Changes in this file all relate to using `Party` rather than `String` as a method argument